Resources/Audio/Voice/Diona/attributions.yml

```diff
@@ -1,4 +1,4 @@
 ﻿- files: ["diona_scream.ogg"]
-  license: "CC-BY-SA-3.0"
-  copyright: "ParadiseSS13"
-  source: "https://github.com/ParadiseSS13/Paradise/blob/82af15e2efd75b566b2280e752b3a4dafbb67ff7/sound/voice/dionatalk1.ogg"
+  license: "CC-BY-4.0"
+  copyright: "InspectorJ. Noise reduced, cropped, converted to mono"
```

Comment:
To better fit how the author would like to be attributed as remarked in the description of the website, phrase it like this:
`Made by InspectorJ (http://www.jshaw.co.uk/) of freesound.org. Modified by Morb0 for SS14 with the following modifications: Noise reduced, cropped, converted to mono`

Also check in-game that the dionas arent too loud when talking because checking the raw file its a lil loud (a video with audio in the description would be nice but i wont require you to)

Reply:
Update and checked in game, not louder than a normal scream, that's for sure.